src/mmw/js/src/account/views.js

```diff
@@ -13,7 +15,63 @@ var Clipboard = require('clipboard'),
     accountTmpl = require('./templates/account.html');
 
 var ProfileView = Marionette.ItemView.extend({
-    template: profileTmpl
+    template: profileTmpl,
+
+    ui: {
+        firstName: '#first_name',
+        lastName: '#last_name',
+        organization: '#organization',
+        userType: '#user_type',
+        country: '#country',
+        postalCode: '#postal_code',
+        saveChanges: '[data-action="save_changes"]'
+    },
+
+    events: {
+        'click @ui.saveChanges': 'saveChanges'
+    },
+
+    initialize: function(){
+        this.listenTo(this.model, 'change', this.render);
+    },
+
+    onRender: function() {
+        var self = this;
+        function addOptions(model, field, selectedValue) {
```

Comment:
For stylistic consistency, prefer defining this as a variable:

```js
var self = this,
    addOptions = function(model, field, selectedValue) {
        ...
    };
```

Reply:
Addressed in fixup 2259d26